src/tactic/core.lean

```diff
@@ -107,6 +107,23 @@ meta def kreplace (e old new : expr) (md := semireducible) (unify := tt)
 
 end expr
 
+namespace name
+
+/-- Auxiliary function for `name.contains_sorry`. -/
+meta def contains_sorry_aux (pre : name) : name → tactic bool | nm := do
```

Comment:
It took me a couple of seconds to understand the type wasn't `(tactic bool) | nm` where `|` and `nm` are things I didn't know. So perhaps:
```suggestion
meta def contains_sorry_aux (pre : name) : name → tactic bool
| nm := do
```

Reply:
this is a very common pattern in the tactic folder, so I think I'll just leave it like this.